examples/intermediate/01_structured_output_simple.py

```
@@ -0,0 +1,129 @@
+    try:
+        # JSON 파싱
+        return json.loads(reply.text)
+    except json.JSONDecodeError:
+        # JSON이 아닌 경우 텍스트에서 JSON 부분 추출 시도
+        text = reply.text
+        start = text.find('{')
+        end = text.rfind('}') + 1
+        if start >= 0 and end > start:
+            try:
+                return json.loads(text[start:end])
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)